libs/hdf-converters/sample_jsons/attestations/rhel7-json-attestedjson.json

```diff
@@ -7999,8 +7999,22 @@
               "start_time": "2019-11-04T16:17:15-05:00",
               "resource": "",
               "skip_message": "File '/etc/audisp/audisp-remote.conf' cannot be found. This check must be performed manually"
+            },
+            {
+              "code_desc": "Manually verified status provided through attestation",
+              "status": "passed",
+              "message": "Attestation:\nStatus: passed\nExplanation: The operating system off-loads audit records onto a different system from the system being audited.\n\nUpdated: 2024-03-12\nUpdated By: Bobby Smalls, Security\nFrequency: 7w",
+              "start_time": "2024-03-12T15:51:51.580Z"
             }
-          ]
+          ],
+          "attestation_data": {
+            "control_id": "V-72083",
+            "explanation": "The operating system off-loads audit records onto a different system from the system being audited.",
+            "frequency": "7w",
+            "status": "passed",
+            "updated": "2024-03-12",
```

Comment:
How do we want to handle these tests - would they fail in 8 weeks, for example? Because of the date?

Reply:
The ones that aren't meant to expire put the current date in the updated field:     
updated: moment().format('YYYY[-]MM[-]DD')

But now that I think about it, the files the tests write to with writefilesync (those commented out lines) won't have the updated dates unless they are uncommented and made to run again. Realistically the answer to that would be to uncomment the lines and have them run every time (which wouldn't affect anything so i'm not sure why they're commented out anyhow)